addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java

```diff
@@ -106,7 +97,7 @@
     private static Set<String> getSupportedVoices() {
```

Comment:
I am aware that you did not introduce this, but isn't this complete none-sense?
It adds the very same string multiple once x-times (where x is the number of locales) to a Set, which will in the end have a single entry (as all added elements are equal). Would you agree that we can remove this method and simply use a singleton list instead?